onnxruntime/python/onnxruntime_pybind_state.cc

```diff
@@ -1502,9 +1502,7 @@ void CreateInferencePybindStateModule(py::module& m) {
   addSparseTensorMethods(m);
   addIoBindingMethods(m);
 
-#if !defined(__APPLE__) && \
-    (!defined(ORT_MINIMAL_BUILD) || defined(ORT_EXTENDED_MINIMAL_BUILD) || defined(ORT_MINIMAL_BUILD_CUSTOM_OPS))
-  Ort::SessionOptions tmp_options;
+#if !defined(__APPLE__) && !defined(ORT_MINIMAL_BUILD)
```

Comment:
updated this to match the condition here:
https://github.com/microsoft/onnxruntime/blob/49c8bbe8768c6897d85b57c12484e72b53fe4831/cmake/onnxruntime_session.cmake#L10